fastify.d.ts

```diff
@@ -144,7 +144,7 @@ export type FastifyServerOptions<
   return503OnClosing?: boolean,
   ajv?: {
     customOptions?: AjvOptions,
-    plugins?: Function[]
+    plugins?: (Function | [Function, string | string[]])[]
```

Comment:
It is the plugin options and we should not expect it to be `string` or `string[]` only.
```suggestion
    plugins?: (Function | [Function, unknown])[]
```

Reply:
I totally agree, thanks.